src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -37,14 +37,8 @@ type Props = {
     onVersionPromote: VersionActionCallback,
     onVersionRestore: VersionActionCallback,
     parentName: string,
+    showUpsellExperience: boolean,
```

Comment:
Is this still needed or can we use the existence of `onUpgradeClick` to determine if the static sidebar should be shown?

Reply:
@jstoffan onUpgradeClick is being passed as `onUpgradeClick: showUpsellExperience && onUpgradeClick` so I do think that our safeguard would still apply. this is a good simplification but do you think this syntax is clear enough for a user (not having the showUpsellExperience explicitly and just using onUpgradeClick)? if so can make the updates!

Reply:
Yeah, I think it should be fine. We're likely the only ones using these props and `onUpgradeClick` isn't used for anything else currently. Best to keep the API as small as possible, I think.